bindgen-cli/options.rs

```diff
@@ -225,6 +225,24 @@ where
                     "Create PartialEq implementation, if it can not be derived \
                      automatically.",
                 ),
+            Arg::new("with-derive-custom")
+                .long("with-derive-custom")
+                .help(
+                    "Derive custom traits on any type. \
+                     Don't use with --with-derive-custom-struct.",
+                )
+                .value_name("trait")
+                .multiple_occurrences(true)
+                .number_of_values(1),
+            Arg::new("with-derive-custom-struct")
```

Comment:
Because I didn't need those, and they caused huge issues with the Derive I was trying to use.
The support is there in the base project, I was implementing this because I need this on the cli.